pages/en/indexing/operating-graph-node.mdx

```diff
@@ -0,0 +1,397 @@
+
+#### Account-like optimisation
+
+Database tables that store entities seem to generally come in two varieties: 'transaction-like', where entities, once created, are never updated, i.e., they store something akin to a list of financial transactions, and 'account-like' where entities are updated very often, i.e., they store something like financial accounts that get modified every time a transaction is recorded. Account-like tables are characterized by the fact that they contain a large number of entity versions, but relatively few distinct entities. Often, in such tables the number of distinct entities is 1% of the total number of rows (entity versions)
+
+For account-like tables, `graph-node` can generate queries that take advantage of details of how Postgres ends up storing data with such a high rate of change, namely that all of the versions for recent blocks are in a small subsection of the overall storage for such a table.
+
+The command `graphman stats show <sgdNNNN`> shows, for each entity type/table in a deployment, how many distinct entities, and how many entity versions each table contains. That data is based on Postgres-internal estimates, and is therefore necessarily imprecise, and can be off by an order of magnitude. A `-1` in the `entities` column means that Postgres believes that all rows contain a distinct entity.
+
+In general, tables where the number of distinct entities are less than 1% of the total number of rows/entity versions are good candidates for the account-like optimization. When the output of `graphman stats show` indicates that a table might benefit from this optimization, running `graphman stats show <sgdNNN> <table>` will perform a full count of the table - that can be slow, but gives a precise measure of the ratio of distinct entities to overall entity versions.
+
+Once a table has been determined to be account-like, running `graphman stats account-like <sgdNNN>.<table>` will turn on the account-like optimization for queries against that table. The optimization can be turned off again with `graphman stats account-like --clear <sgdNNN>.<table>` It takes up to 5 minutes for query nodes to notice that the optimization has been turned on or off. After turning the optimization on, it is necessary to check in the appropriate dashboard that the change does not in fact make queries slower for that table; usually, slower queries cause queries against that table to show up in `pg_stat_activity` ([here](https://thegraph.grafana.net/d/Mo6FxoiWz/p-postgres-stats-prod?orgId=1&var-shard=graph_evm_compat&var-autovacuum=xautovacuum&viewPanel=4)) in large numbers, taking several seconds. In that case, the optimization needs to be turned off again.
+
+For Uniswap-like subgraphs, the `pair` and `token` tables are prime candidates for this optimization, and can have a dramatic effect on database load.
+
+#### Creating new indices
+
+> TODO: specific examples on index changes which can help with slow queries
```

Comment:
@lutter any pointers that are worth including here?

Reply:
I need to write something up :(

Reply:
I think as this isn't released yet I have removed this section for now